resources/assets/js/videos/components/thumbnailPreview.vue

```diff
@@ -101,12 +100,9 @@ export default {
         hoverTimeStyle() {
             return { 'font': `bold ${this.fontSize}px Sans-Serif`, 'color': '#cccccc' };
         },
-        xtext() {
-            return this.canvasWidth / 2;
-        },
-        ytext() {
-            return this.canvasHeight + this.fontSizeInPx + (this.hoverTimeBarHeight - this.fontSizeInPx) / 2;
-        },
+        fontSizeInPx() {
+            return this.fontSize * (72 / 96);
```

Comment:
At least we can save this computation :wink: 
```suggestion
            return this.fontSize * 0.75;
```